packages/shared/src/components/search/SearchBarSuggestionList.tsx

```diff
@@ -31,7 +31,7 @@ export function SearchBarSuggestionList({
     return (
       <PlaceholderSearchSuggestion
         className={className}
-        onClick={() => showLogin(LoginTrigger.SearchSuggestion)}
```

Comment:
@rebelchris just checking with you since you added similar comment for Origin, should this (and the other usages) be left as LoginTrigger?

Reply:
This should be fine as it's a LoginTrigger, just don't want to mix Origin and LoginTrigger for different purposes

Reply:
Cool then it's ok to leave these now as AuthTriggers instead of LoginTriggers.